website/content/docs/api-gateway/tech-specs.mdx

```diff
@@ -42,6 +42,16 @@ The following table describes the TCP port requirements for each component of th
   - Self-Managed
   - HCP Consul
 
+### Limited Support of some Consul Features
+
+The following table lists API Gateway limitations related to specific Consul features
+
+| Consul Feature | Limitation |
+| -------------- | ---------- |
+| Admin Partitions | API Gateway can deployed only in the default Admin Partition. |
+| Datacenter Federation | When multiple Consul datacenters are federated together, API Gateway can only be deployed in the primary datacenter. |
+| Routing between Datacenters |  When multiple Consul datacenters are federated together, API Gateway can route traffic only to Services in the local datacenter; however, API Gateway can route to Services in other Kubernetes clusters when they are in the same Consul datacenter. See [Single Consul Datacenter in Multiple Kubernetes Clusters](https://www.consul.io/docs/k8s/deployment-configurations/single-dc-multi-k8s) for more details. |
```

Comment:
```suggestion
| [Admin partitions](/docs/enterprise/admin-partitions) | You can only deploy Consul API Gateway into the `default` admin partition. |
| Datacenter federation | If you are connecting multiple Consul datacenters to create a federated network, you can only deploy Consul API Gateway to the primary datacenter. |
| Routing between datacenters |  If you are connecting multiple Consul datacenters to create a federated network, API Gateway can only route traffic to Kubernetes Services in the local datacenter. However, Consul API Gateway can route to Services in other Kubernetes clusters when they are in the same Consul datacenter. Refer to [Single Consul Datacenter in Multiple Kubernetes Clusters](/docs/k8s/deployment-configurations/single-dc-multi-k8s) for more details. |
```
I don't know that I would have formatted this information into a table, but I can understand the decision. 
The main feedback is to use active voice and proper capitalization per the style guide - I believe we agreed that "Consul API Gateway" is a proper noun, but "the API gateway" is not. Capital "S" Services refer to Kubernetes Services (because that's how they write about them) and distinguished from "s" services registered with Consul. 
The only other thing to call out is the way we refer to the concept of federation. I don't think we need to explain it every time, but I wonder if there is room to be more concise so that people who are unfamiliar with the term (however unlikely) have a little more context. Just a note to myself, we should add it as a glossary term so that we have something to link to.

Reply:
```suggestion
| [Admin partitions](/docs/enterprise/admin-partitions) | You can only deploy Consul API Gateway into the `default` admin partition. |
| Datacenter federation | If you are connecting multiple Consul datacenters to create a federated network, you can only deploy Consul API Gateway in the `primary` datacenter. |
| Routing between datacenters | If you are connecting multiple Consul datacenters to create a federated network, API Gateway can only route traffic to Services in the local datacenter. However, API Gateway can route to Services in other Kubernetes clusters when they are in the same Consul datacenter. Refer to [Single Consul Datacenter in Multiple Kubernetes Clusters](/docs/k8s/deployment-configurations/single-dc-multi-k8s) for more details. |
```
Adam, Thanks for converting this to active voice. You are correct that "API gateway" is not a proper noun when used to refer to a type of gateway. It is a proper noun when being used as a shortened version of the product name "Consul API Gateway".